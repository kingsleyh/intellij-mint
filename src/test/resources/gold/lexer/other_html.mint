component Super {

  fun render : Html {
    <div::rangeSlider>
    <div><{effortScale(Number.fromString(sliderPos) |> Maybe.withDefault(0))}></div>
      <input::sliderRange
        value={sliderPos}
        step={Number.toString(step)}
        max={Number.toString(max)}
        min={Number.toString(min)}
        disabled={disabled}
        onChange={changed}
        type="range"/>

      <span::rangeValue>
        <{ sliderPos }>
      </span>
    </div>
  }

}