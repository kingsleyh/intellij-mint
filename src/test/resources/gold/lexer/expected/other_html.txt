component ('component')
WHITE_SPACE (' ')
IDENT ('Super')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('render')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Html')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
< ('<')
IDENT ('div')
:: ('::')
IDENT ('rangeSlider')
> ('>')
WHITE_SPACE ('\n    ')
< ('<')
IDENT ('div')
> ('>')
< ('<')
{ ('{')
IDENT ('effortScale')
( ('(')
IDENT ('Number')
. ('.')
IDENT ('fromString')
( ('(')
IDENT ('sliderPos')
) (')')
WHITE_SPACE (' ')
|> ('|>')
WHITE_SPACE (' ')
IDENT ('Maybe')
. ('.')
IDENT ('withDefault')
( ('(')
NUMBER_LITERAL ('0')
) (')')
) (')')
} ('}')
> ('>')
</ ('</')
IDENT ('div')
> ('>')
WHITE_SPACE ('\n      ')
< ('<')
IDENT ('input')
:: ('::')
IDENT ('sliderRange')
WHITE_SPACE ('\n        ')
IDENT ('value')
= ('=')
{ ('{')
IDENT ('sliderPos')
} ('}')
WHITE_SPACE ('\n        ')
IDENT ('step')
= ('=')
{ ('{')
IDENT ('Number')
. ('.')
IDENT ('toString')
( ('(')
IDENT ('step')
) (')')
} ('}')
WHITE_SPACE ('\n        ')
IDENT ('max')
= ('=')
{ ('{')
IDENT ('Number')
. ('.')
IDENT ('toString')
( ('(')
IDENT ('max')
) (')')
} ('}')
WHITE_SPACE ('\n        ')
IDENT ('min')
= ('=')
{ ('{')
IDENT ('Number')
. ('.')
IDENT ('toString')
( ('(')
IDENT ('min')
) (')')
} ('}')
WHITE_SPACE ('\n        ')
IDENT ('disabled')
= ('=')
{ ('{')
IDENT ('disabled')
} ('}')
WHITE_SPACE ('\n        ')
IDENT ('onChange')
= ('=')
{ ('{')
IDENT ('changed')
} ('}')
WHITE_SPACE ('\n        ')
IDENT ('type')
= ('=')
STRING_LITERAL ('"range"')
/ ('/')
> ('>')
WHITE_SPACE ('\n\n      ')
< ('<')
IDENT ('span')
:: ('::')
IDENT ('rangeValue')
> ('>')
WHITE_SPACE ('\n        ')
< ('<')
{ ('{')
WHITE_SPACE (' ')
IDENT ('sliderPos')
WHITE_SPACE (' ')
} ('}')
> ('>')
WHITE_SPACE ('\n      ')
</ ('</')
IDENT ('span')
> ('>')
WHITE_SPACE ('\n    ')
</ ('</')
IDENT ('div')
> ('>')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')